docs/TPM2_PCR_MEASUREMENTS.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Recognizing TPM2 PCR Measurements Made by systemd
+category: Booting
+layout: default
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Recognizing TPM2 PCR Measurements Made by systemd
+
+Various systemd components issue TPM2 PCR measurements during the boot process,
+both in UEFI mode and from userspace. The following lists all measurements
+done, and describes (in case done before `ExitBootServices()`) how they appear
+in the TPM2 Event Log, maintained by the PC firmware.
+
+systemd will measure to PCRs 11, 12, 13, 15.
```

Comment:
We now have official names for the PCRs, so I think this should say
"foobar (11), barbar (12), …."